Makefile

```diff
@@ -169,7 +169,7 @@ test-sim-benchmark-invariants:
 	-Period=1 -Commit=true -Seed=57 -v -timeout 24h
 
 cli-test: build-sim
-	@go test -mod=readonly -p 4 `go list ./tests/cli/tests/...` -tags=cli_test -v
+	@go test -mod=readonly -p 4 `go list ./tests/cli/...` -tags=cli_test -v
 	@go test -mod=readonly -p 4 `go list ./x/.../client/cli_test/...` -tags=cli_test -v
```

Comment:
```suggestion
	go test -mod=readonly -p 4 `go list ./x/.../client/cli_test/...` -tags=cli_test -v -builddir=$(BUILDDIR)
```

Reply:
I'd remove all the `@` characters at the beginning of commands' lines in Make. It's useful when debugging to see exactly what commands are being executed.